src/pflow/pflow.cpp

```diff
@@ -460,7 +488,7 @@ PetscErrorCode PFLOWFunction(SNES snes, Vec X, Vec F, void *ctx) {
         farr[loc] -= gen->pg;
         flps += 1;
         if (bus->ide == PV_BUS) {
-          farr[loc + 1] = Vm - gen->vs;
+          farr[loc + 1] += Vm - gen->vs;
```

Comment:
Changes like this are eyebrow raisers, but I trust your judgement here

Reply:
Yeah, this change was necessary for multiple generators at the same bus.